packages/@vue/cli-plugin-router/generator/template/src/App.vue

```diff
@@ -0,0 +1,65 @@
+---
+extend: '@vue/cli-service/generator/template/src/App.vue'
+replace:
+  - !!js/regexp /<template>[^]*?<\/template>/
+  - !!js/regexp /<script>[^]*?<\/script>/
```

Comment:
This will still introduce 2 extraneous blank lines. Could be eliminated by changing the regexp to
```suggestion
  - !!js/regexp /\n<script>[^]*?<\/script>\n/
```